src/py/flwr/server/app.py

```diff
@@ -681,3 +698,52 @@ def _add_args_fleet_api(parser: argparse.ArgumentParser) -> None:
         type=int,
         help="Set the number of concurrent workers for the Fleet API server.",
     )
+
+
+def _add_args_exec_api(parser: argparse.ArgumentParser) -> None:
+    """Add command line arguments for Exec API."""
+    parser.add_argument(
+        "--exec-api-address",
+        help="Exec API server address (IPv4, IPv6, or a domain name)",
+        default=EXEC_API_DEFAULT_ADDRESS,
+    )
+    parser.add_argument(
+        "--executor",
+        help="For example: `deployment:exec` or `project.package.module:wrapper.exec`. "
+        "The default is `flwr.superexec.deployment:executor`",
+        default="flwr.superexec.deployment:executor",
+    )
+    parser.add_argument(
+        "--executor-dir",
+        help="The directory for the executor.",
+        default=".",
+    )
+    parser.add_argument(
+        "--executor-config",
+        help="Key-value pairs for the executor config, separated by spaces. "
+        "For example:\n\n`--executor-config 'verbose=true "
+        'root-certificates="certificates/superlink-ca.crt"\'`',
+    )
+
```

Comment:
Do we need these other args for secure connection? 
```python
    parser.add_argument(
        "--executor-ssl-certfile",
        help="Exec API server SSL certificate file (as a path str) "
        "to create a secure connection.",
        type=str,
        default=None,
    )
    parser.add_argument(
        "--executor-ssl-keyfile",
        help="Exec API server SSL private key file (as a path str) "
        "to create a secure connection.",
        type=str,
    )
    parser.add_argument(
        "--executor-ssl-ca-certfile",
        help="Exec API server SSL CA certificate file (as a path str) "
        "to create a secure connection.",
        type=str,
    )    
```

Reply:
I don't think we need an extra tuple of certificates specifically for `Exec API`. These certificates allow the SuperLink to estabilish an SSL-enabled connection with a grpc client, and we can just reuse the certificates for `Fleet API` and `Driver API`.